schema/test/unit/util_functions/get_form_status_test.sql

```diff
@@ -109,6 +109,27 @@ select results_eq(
   'Returns Not Started when new form data is null and form change is pending'
 );
 
+update cif.form_change set new_form_data='{}', change_status='staged'
+  where project_revision_id=1
+    and form_data_table_name='project_contact';
+
+/* test 7 */
+
+update cif.form_change set operation='archive', change_status='staged'
+  where project_revision_id=1
+    and validation_errors != '[]'
+    and form_data_table_name= 'project_contact';
```

Comment:
I had a little trouble following what the test data looked like at this point. I think it might be clearer to do something like test 5 and set everything including errors just before the assertion

Reply:
I'll try and make it clearer